target/linux/mpc85xx/files/arch/powerpc/boot/dts/tl-wdr4900-v1.dts

```diff
@@ -118,6 +118,23 @@
 		usb@22000 {
 			phy_type = "utmi";
 			dr_mode = "host";
+
+			port@1 {
+				#address-cells = <1>;
+				#size-cells = <0>;
+				reg = <1>;
+				#trigger-source-cells = <0>;
+
+				hub_port1: port@1 {
```

Comment:
normally regs start at 0. Hopefully this is OK.

Reply:
Yes, both leds work.